types/aggregator.go

```diff
@@ -2,11 +2,16 @@ package types
 
 import "time"
 
+const (
+	DefaultAggregatorVersion = "v0.1.12"
+)
+
 // Aggregator model.
 type Aggregator struct {
 	ID             string    `json:"id" yaml:"id"`
 	Token          string    `json:"token" yaml:"token"`
 	Name           string    `json:"name" yaml:"name"`
+	Version        string    `json:"version" yaml:"version"`
```

Comment:
Is it required?

What will we do for the existing aggregators?
Fill a default value during migragion?

Reply:
yep, will default to DefaultAggregatorVersion

Reply:
Can you update the spce then and add it to the `required` list then?

Reply:
done @nicolasparada